src/snarky/index.js

```diff
@@ -0,0 +1,30 @@
+import { getSnarky, snarky_ready, shutdown } from './wrapper';
+import snarkySpec from './snarky-class-spec.json';
+import { proxyClasses, proxyFunctions } from './proxy.js';
+
+export {
+  Field,
+  Bool,
+  Circuit,
+  Poseidon,
+  Group,
+  Scalar,
+  Ledger,
+  shutdown,
+  isReady,
+  picklesCompile,
+};
+
+let isReadyBoolean = false;
+let isReady = snarky_ready.then(() => (isReadyBoolean = true));
+let isItReady = () => isReadyBoolean;
```

Comment:
Why do we declare a function that returns `isReadyBoolean` instead of using it directly?

Reply:
Because we need to access this at later points in time, when the value might have changed!